velox/functions/lib/Re2Functions.cpp

```diff
@@ -149,8 +149,13 @@ bool re2Extract(
       result.setNoCopy(row, StringView(extracted.data(), extracted.size()));
       return !StringView::isInline(extracted.size());
     } else {
-      result.setNull(row, true);
-      return false;
+      if (emptyNoMatch) {
```

Comment:
Can we document this parameter perhaps above L121?

Reply:
Just added a comment at the place where `emptyNoMatch_` is declared.